packages/orbit-components/src/LinkList/__tests__/index.test.tsx

```diff
@@ -3,10 +3,10 @@ import * as React from "react";
 import { screen, render } from "../../test-utils";
 import LinkList from "..";
 
-describe("#LinkList", () => {
+describe("LinkList", () => {
   it("should have expected dom output", () => {
     render(
-      <LinkList legacy dataTest="kek">
+      <LinkList dataTest="kek">
```

Comment:
This test should not change. The `legacy` prop should still be tested that it works as expected (either that or remove the test entirely, as we are asserting styles)